### ISSUE TITLE
Translation into other languages

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/The_Earths_Gut#22]
I've been maintaining a potfile for a bit, eventually I'll want to see if I can get some other people to submit translations for it... kinda depends on #7 a bit...

**Comment by cooljeanius** [comment]
(the difference between this and #6 is that that was just about getting the potfile to generate, while this is about actually translating)

**Comment by cooljeanius** [comment]
@k-konami sent me a Japanese translation that I committed in [6cb0890](https://github.com/cooljeanius/The_Earths_Gut/commit/6cb08903bd06a09f63ea08a8dedfc96df2f6d5cd); there's some additional feedback that I still need to address, though

**Comment by cooljeanius** [comment]
Maybe @celerini and/or @MultiSeptim can help with Czech?

**Comment by MultiSeptim** [comment]
Maybe in the future, but certainly not this year. I have other projects to complete, mainly Winds of Fate.

**Comment by cooljeanius** [comment]
> Maybe in the future, but certainly not this year. I have other projects to complete, mainly Winds of Fate.

ok that's fair

**Comment by celerini** [comment]
not now, but thanks for an offer

**Comment by cooljeanius** [comment]
> not now, but thanks for the offer

ok, thanks for letting me know; the text will probably still be going through a few more revisions anyways, so it might be better to wait in any case...

**Comment by cooljeanius** [comment]
@k-konami could you take another look at this please?

**Comment by k-konami** [comment]
@cooljeanius 
> @k-konami could you take another look at this please?

I have updated TEG.pot locally and also updated ja.po, but I have not updated this week's (0.5.1) version yet. 
I think I will probably just update a couple of places.
Could you update TEG.pot first?

Also, it looks like TEG.pot.diff hasn't been updated for a while, 
do you plan to update it anymore in the future?

**Comment by k-konami** [comment]
>[translation updates](https://github.com/cooljeanius/The_Earths_Gut/commit/286b3b478c103483fdebe00be79cd8a6d522f9c5)

Thank you!!

**Comment by k-konami** [comment]
@cooljeanius 
(I may be mistaken, though...)
Is the following statement correct?
Can you check?
(To me, it reads as if the direction of level advancement for each unit is reversed.)

S08 .cfg :  line 1549 (  TEG.pot id = 416 )
`in addition to Krolock. Remaining available Trolls, Troll Warriors, Boulderlobbers or Rocklobbers can be Troll Whelps, remaining Troll Fire Wizards, Troll Shamans, Troll Frost Wizards or Ice Trolls can be Apprentice Shamans (if there are enough). Remaining Troll Whelps can't be transferred to Apprentice Shamans or vice versa. You will be able to choose the units that stay."`

**Comment by cooljeanius** [comment]
> @cooljeanius (I may be mistaken, though...) Is the following statement correct? Can you check? (To me, it reads as if the direction of level advancement for each unit is reversed.)
> 
> S08 .cfg : line 1549 ( TEG.pot id = 416 ) `in addition to Krolock. Remaining available Trolls, Troll Warriors, Boulderlobbers or Rocklobbers can be Troll Whelps, remaining Troll Fire Wizards, Troll Shamans, Troll Frost Wizards or Ice Trolls can be Apprentice Shamans (if there are enough). Remaining Troll Whelps can't be transferred to Apprentice Shamans or vice versa. You will be able to choose the units that stay."`

Hm... @Konrad22 WDYT? (Since I'm working on changing this for issue #42 anyways...)

**Comment by Konrad22** [comment]
Maybe just change 'Remaining available...can be' to 'Remaining space for...can be filled with/used by' for clarification?
And in that case also 
Remaining Troll Whelps can't be transferred to Apprentice Shamans -> Remaining space for Troll Whelps can't be transferred to Apprentice Shamans

**Comment by k-konami** [comment]
For the time being, I will pull-req the current latest translation.

And to @cooljeanius, thank you (though late) for the thoughtful explanations and answers you added to the comments I had written on ja.po. They were very helpful.

**Comment by cooljeanius** [comment]
Czech translation merged in c04713e; maybe @Toranks can do a Spanish translation next?

**Comment by cooljeanius** [comment]
> Czech translation merged in [c04713e](https://github.com/cooljeanius/The_Earths_Gut/commit/c04713e84e245c079c9da9d8c8b9d121fbbcfc21); maybe [@Toranks](https://github.com/Toranks) can do a Spanish translation next?

...or maybe @nadhem000 can do an Arabic one?
